wikiclass/utilities/fetch_labels.py

```diff
@@ -82,12 +80,8 @@ def aggregate_labels(label_docs):
         return None
     else:
         if sum('automated' not in l['data'] for l in label_docs) > 0:
-            relevant_label_docs = (l for l in label_docs
-                                   if 'automated' not in l['data'])
             auto_labeled = False
         else:
-            relevant_label_docs = (l for l in label_docs
-                                   if 'automated' in l['data'])
```

Comment:
Seems like this should not be removed.

Reply:
It's not used at all (unused variable)

Reply:
Oh my.  This script is super messed up.  Looks like it was something someone else put together.  I'm merging this but we should fix this script.